components/content-service/pkg/initializer/git.go

```diff
@@ -186,7 +194,7 @@ func (ws *GitInitializer) realizeCloneTarget(ctx context.Context) (err error) {
 		}
 
 		// checkout specific commit
-		if err := ws.Git(ctx, "checkout", ws.CloneTarget); err != nil {
+		if err := ws.Git(ctx, "switch", "-C", ws.CloneTarget); err != nil {
```

Comment:
Hey @aledbf :wave: ,

question (non-blocking): how come some case statements :point_up: are logging errors and others do not? I ask because I'm not sure why there is variance. 

It seems like we may want to log the errors in each case (especially because we're changing from `checkout` to `switch`), but I could also be being naive. I haven't opened this to see if the calling function logs the related error.

https://github.com/gitpod-io/gitpod/pull/13053/files#diff-c9626ddd8d63f48f858c8e81807c14008f5c4199bd41ba80a2981401111356f0R186

https://github.com/gitpod-io/gitpod/pull/13053/files#diff-c9626ddd8d63f48f858c8e81807c14008f5c4199bd41ba80a2981401111356f0R193

https://github.com/gitpod-io/gitpod/pull/13053/files#diff-c9626ddd8d63f48f858c8e81807c14008f5c4199bd41ba80a2981401111356f0R198